src/native/corehost/fxr/hostfxr.cpp

```diff
@@ -545,6 +545,7 @@ namespace
 
         if (startup_info.host_path.empty())
         {
+            // Use realpath to find the host_path behind symlinks
```

Comment:
Should we also "realpath" the `host_path` if it is specified by the caller? This comes from custom host code (so external input).
I think for consistency it would make sense to follow symlinks everywhere, but it is technically a breaking change - and in hostfxr :-(